inspire_utils/name.py

```diff
@@ -196,6 +199,16 @@ def _is_roman_numeral(suffix):
 
         return final_name
 
+    @classmethod
+    def from_parts(cls, first, last, middle=None, suffix=None, title=None):
```

Comment:
you should handle also the `constants` in this case (please add a test), and `first` and `last` should both be optional, you are assuming too much of names of people.